### DISCUSSION TITLE
Welcome to eluceo/iCal Discussions!

### DISCUSSION THREAD

**Discussion by markuspoerschke** [markuspoerschke/iCal#253] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded.

➡️ [Let’s get started](https://github.com/markuspoerschke/ical/discussions/new)

If you want to **report a bug** or **suggest a feature**, please head over to [our issue tracker](https://github.com/markuspoerschke/iCal/issues/new/choose).

**Comment by giorginogreg** [comment]
@markuspoerschke I would like to maintain this repository if you're not anymore active, so that people could still contribute to it. What about to talk about this?

**Reply by 4d4ch4u32** [reply]
Since this repository is apparently no longer maintained, I have forked it.

The source code of the fork is available here: https://git.var-lab.com/opencal/ical. I will continue the development starting with version `3.0.0` (PHP < 8.0 is not longer supported by the forked version). It is also available via packagist.org:

```
composer require opencal/ical
```

Contributions are welcome!
